File: src_front_end/src/services/MstBumonService.js
import http from "../http-common";

const MstBumonService = {
  listDepartmentByStepNumber(stepNumber, mstApprovalflowCd) {
    return http.get(`/mst-bumon?stepNumber=${stepNumber}`);
  }

}

export default MstBumonService;

File: src/main/java/sskf/service/MstBumonService.java
package sskf.service;

import sskf.model.basemodel.BaseSearchRequest;
import sskf.model.response.BumonResponse;

import java.util.List;

public interface MstBumonService {

    List<BumonResponse> listDepartmentForItem(BaseSearchRequest baseSearchRequest);

    List<BumonResponse> listDepartmentByStepNumber(Byte stepNumber);
}
